optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/PlannerBenchmarkRunner.java

```diff
@@ -338,4 +378,23 @@ public long calculateTimeMillisSpent() {
         return now - startingSystemTimeMillis;
     }
 
+    private static final class ConfigBackup {
```

Comment:
ConfigBackup is too general - it doesn't state it's purpose.

I would include the word "warmUp" or even "duringWarmUp" to indicate it's purpose/reason of existence

Some proposals:
- WarmUpConfigBackup
- BenchmarkConfigBackupDuringWarmUp
- OriginalConfigPartsDuringWarmUp
- ...